BDArmory/Ammo/PooledBullet.cs

```diff
@@ -1262,37 +1263,37 @@ bool BulletHitAnalysis(RaycastHit hit, float period)
 
                             // In the case we are reaching that cap we decrease the velocity by
                             // the adjustedPenRatio minus the portion that went into erosion
-                            impactVelocity = impactVelocity * adjustedPenRatio;
-                            currentVelocity = hitPartVelocity + impactVelocity;
+                            //impactVelocity = impactVelocity * adjustedPenRatio;
+                            //currentVelocity = hitPartVelocity + impactVelocity;
                         }
                         else
                         {
                             bulletMass = bulletMass * massRatio;
 
                             // If we don't, I.E. the round isn't completely eroded, we decrease
                             // the velocity by a max of 5%, proportional to the adjustedPenRatio
-                            impactVelocity = impactVelocity * (0.95f + 0.05f * adjustedPenRatio);
-                            currentVelocity = hitPartVelocity + impactVelocity;
+                            //impactVelocity = impactVelocity * (0.95f + 0.05f * adjustedPenRatio);
+                            adjustedPenRatio *= 0.05f;
```

Comment:
Ah, I see I read the code change wrong! I saw the commented out lines
```csharp
//impactVelocity = impactVelocity * (0.95f + 0.05f * adjustedPenRatio);
//impactVelocity = impactVelocity * adjustedPenRatio;
//currentVelocity = hitPartVelocity + impactVelocity;
```
and thought that that was the original code.

Also, by "capped", you mean clamped in the range 0.95—1, right (since that's what this code is doing, e.g., 0.5 → 0.975)?